docs/connector-development/config-based/connector-builder-compatibility.md

```diff
@@ -0,0 +1,210 @@
+
+You can follow [this chart](https://whimsical.com/how-should-i-build-my-connector-YNXaNQhRuPuWy68Xg8Akhy) to guide you through the process.
+
+The sections below provide more details to help you answer the questions.
+
+## Is the integration a REST API returning a collection of records synchronously?
+
+Is the integration a REST API? In doubt, look for the word “REST” in the documentation.
+
+Rook at the shape of the resource path.
+
+Taking the [Congress API](https://api.congress.gov/#/bill) as an example,
+
+`GET /bill`
+
+Indicates the records can be retrieved by submitting a GET request to the /bill path.The sample response shows that the response returns a collection of records, so the Congress API is REST API returning a collection of records.
+
+Sample response:
+```
+{
+  "bills":[
+    {
+      "congress":117,
+      "latestAction":{
+        "actionDate":"2022-04-06",
+        "text":"Became Public Law No: 117-108."
+      },
+      "number":"3076",
+      "originChamber":"House",
+      "originChamberCode":"H",
+      "title":"Postal Service Reform Act of 2022",
+      "type":"HR",
+      "updateDate":"2022-09-29",
+      "updateDateIncludingText":"2022-09-29T03:27:05Z",
+      "url":"https://api.congress.gov/v3/bill/117/hr/3076?format=json"
+    },
+    {
+      "congress":117,
+      "latestAction":{
+        "actionDate":"2022-04-06",
+        "text":"Read twice. Placed on Senate Legislative Calendar under General Orders. Calendar No. 343."
+      },
+      "number":"3599",
+      "originChamber":"House",
+      "originChamberCode":"H",
+      "title":"Federal Rotational Cyber Workforce Program Act of 2021",
+      "type":"HR",
+      "updateDate":"2022-09-29",
+      "updateDateIncludingText":"2022-09-29T03:41:50Z",
+      "url":"https://api.congress.gov/v3/bill/117/hr/3599?format=json"
+    }
+  ]
+}
+```
+
+Some endpoints are parameterized. An example of such endpoint is the [Congress API’s bill by congress endpoint](https://api.congress.gov/#/bill/bill_list_by_congress).
+`GET /bill/:congress`
+
+These endpoints are also valid synchronous REST endpoints.
+
+This differs from the [Amazon Ads reports endpoint](https://advertising.amazon.com/API/docs/en-us/info/api-overview)), which returns a report ID, which will be generated asynchronously by the source. Because the reports need to be downloaded separately, this is not a synchronous REST API.
+
+Examples:
+- Yes: [Congress API](https://api.congress.gov/#/)
+- No: [Amazon Ads](https://advertising.amazon.com/API/docs/en-us/info/api-overview)
+
+If the integration is not a synchronous REST API, use the Python CDK.
+
+## What type of authentication is required?
+Look up the authentication mechanism in the API documentation, and identify which type it is.
+
+### Basic HTTP
+Are requests authenticated using the Basic HTTP authentication method? You can search the documentation page for one of the following keywords
```

Comment:
These section just ask yes/no questions, without indicating what the answer means in terms of compatibility with the builder.

After each of these questions are asked, the next sentence should say something like `If yes, then that is compatible with the Builder. If no, use the Python CDK.`